app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
+
+    return counter
```

Comment:
The solution is ok, but can you try to rewrite it using just 1 line?

Reply:
I rewrited my code, but accidentally close my pull request. Can you please chek my new commit?